cmd/graphsplit/main.go

```diff
@@ -51,30 +51,35 @@ var chunkCmd = &cli.Command{
 			Required: true,
 			Usage:    "specify graph name",
 		},
-		&cli.StringFlag{
-			Name:  "parent-path",
-			Value: "",
-			Usage: "specify graph parent path",
-		},
```

Comment:
I want to keep this feature, because the following functions cannot be implemented without this parameter. For example, if I want to split a subdirectory of a directory, but I also want to keep its root directory, I can do this:

./graphsplit chunk \
--car-dir=path/to/car-dir \
--slice-size=17179869184 \
--parallel=2 \
--graph-name=gs-test \
--parent-path=/path/to/dataset \
/path/to/dataset/subdir

What would I do without this feature? Can we set a default parameter to make --parent-path the same as the directory we specified for the split?

Reply:
Got it.